spmd/tensor/api.py

```diff
@@ -66,6 +67,21 @@ def __repr__(self):
         # TODO: consider all_gather the local tensors for better debugging
         return f"DistributedTensor({self._local_tensor}, placements={self._placements})"
 
+    @classmethod
```

Comment:
nit: rebase with main?

Reply:
I did but your change somehow gets into this PR so the plan is that when I merge this PR to master. I will manually handle the merge conflict there. Next time, I will merge rather than rebase... (I get used to rebase too much).